### PR TITLE
feat: add etherscan tutorial

### PR DESCRIPTION
**Pull request by maxdesalle**

### CODE REVIEW

**Comment by maxdesalle** [comment]
I didn't include a section about token approvals, should this be added @razgraf? Don't know if all tokens have the same approval mechanism, so not sure if we can provide a tutorial that will work for every token out there.

**Comment by razgraf** [comment]
Amazing Max! Great work on the tutorial.

> I didn't include a section about token approvals, should this be added @razgraf? Don't know if all tokens have the same approval mechanism, so not sure if we can provide a tutorial that will work for every token out there.

Yes, we should! But I can take care of it tomorrow, as I review the other bits.

**Comment by razgraf** [comment]
Within the last commit I've added:

- an allowance/erc20-approve section on how to perform it on etherscan
Allowance
- code examples (so we can also easily copy-paste things)
Screenshot 2023-08-11 at 18 53 47
- refactored the page into steps so we can extend the right-side content tree

**Review by razgraf — APPROVED** [review]

**Review by PaulRBerg — COMMENTED** [review]
just wanted to say that this is great work, thanks @maxdesalle!

File: docs/contracts/v2/guides/08-etherscan.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+### Prerequisites
+
+Before being able to create a stream using the Sablier V2 Core contracts you need to grant the right allowance. See the
+[Allowances](#allowances) section for a guide on how to do that.
+
+### Step 1: Go to contract page
+
+Head over to our [deployments](/contracts/v2/deployments) list and select the contract address you want to interact
+with. For every chain, that will either be `SablierV2LockupLinear` or `SablierV2LockupDynamic`. In this tutorial, we
+will show how to create a stream using **Linear** variant on Goerli.
```
Comment (PaulRBerg) [inline docs/contracts/v2/guides/08-etherscan.md:27]: Can we say **LockupLinear**?

File: docs/contracts/v2/guides/08-etherscan.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+  range: [1704067200,1704067200,1735689600],
+  broker: ["0x0000000000000000000000000000000000000000",0]
+}
+```
+
+#### Sender
+
+The `sender` is the wallet that will have the ability to cancel the stream, if the stream is cancelable. If it's
+non-cancelable, the sender address will keep some limited capabilities (as mapped
+[here](/contracts/v2/reference/access-control#overview)).
```
Comment (PaulRBerg) [inline docs/contracts/v2/guides/08-etherscan.md:74]: The sender actually has no capabilities if the stream is non-cancelable. I will adjust this

**Review by PaulRBerg — COMMENTED** [review]
Why didn't we use `createWithDurations` in the tutorial?

Feels more practical than `createWithRange`, because the examples under "Range" could have actually been used.

File: docs/contracts/v2/guides/08-etherscan.md (hunk 2 above)
Comment (razgraf) [reply docs/contracts/v2/guides/08-etherscan.md:74]: Can't they withdraw on behalf of the sender?

File: docs/contracts/v2/guides/08-etherscan.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+Most users will put their own wallet address there.
+
+#### Recipient
+
+The address you want to stream tokens to. They will be able to [withdraw](#withdrawing-from-a-stream) tokens as they
+become available.
+
+#### Total Amount
+
+The `totalAmount` is the total amount of tokens you want to stream, **DECIMALS INCLUDED**. If the token you are looking
```
[the remark this answers is not in the document]
Comment (razgraf) [reply docs/contracts/v2/guides/08-etherscan.md:85]: I agree we should mention fees are included, but not in the same breath. Should be a separate sentence. The accent here should be on the decimals being necessary - which is in itself a strange topic for newcomers (at least that's how I think Max intended it).

[the remark this answers is not in the document]
Comment (PaulRBerg) [reply docs/contracts/v2/guides/08-etherscan.md:85]: That makes sense - feel free to edit directly on `main`, @razgraf.

File: docs/contracts/v2/guides/08-etherscan.md (hunk 2 above)
Comment (PaulRBerg) [reply docs/contracts/v2/guides/08-etherscan.md:74]: oh right, forgot about withdrawals.

that special section sounds overkill; there are a bunch of ways in which we could categorize the state of the contract. I don't find it particularly helpful to zero in on this dimension of cancelable vs non-cancelable (in "Access Control").

**Pull request merged**